src/Microsoft.AspNet.Server.Kestrel/Filter/SocketInputStream.cs

```diff
@@ -5,6 +5,7 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
+using IllyriadGames.ByteArrayExtensions;
```

Comment:
Want to drop net451 and move to only supporting net46? I've just been made aware of `Buffer.MemoryCopy` which should offer better performance... (Doesn't appear till 4.6 and coreclr)